modules/gdscript/gdscript_editor.cpp

```diff
@@ -3456,6 +3456,30 @@ ::Error GDScriptLanguage::lookup_code(const String &p_code, const String &p_symb
 				return OK;
 			}
 		} break;
+		case GDScriptParser::COMPLETION_TYPE_ATTRIBUTE: {
+			const GDScriptParser::TypeNode *type = static_cast<const GDScriptParser::TypeNode *>(context.node);
+
+			const GDScriptParser::IdentifierNode *prev = nullptr;
+			GDScriptParser::DataType base_type;
+			for (const GDScriptParser::IdentifierNode *E : type->type_chain) {
+				if (E->name == p_symbol) {
+					base_type = prev->get_datatype();
```

Comment:
If this condition would be true on the first iteration then it would crash as `prev` would still be `nullptr`. Is it guaranteed to not happen? If not, needs to be handled properly.

Reply:
I added a check to be safe